accounting/aggregations/query.py

```diff
@@ -0,0 +1,547 @@
+import elasticsearch
+import argparse
+from elasticsearch_dsl import Search, Q, A
+from datetime import datetime, timedelta
+from collections import namedtuple
+from operator import itemgetter
+from pprint import pprint
+
+from report_helpers import *
```

Comment:
`import *` is convenient but usually considered bad practice

```suggestion
from report_helpers import Aggregation, add_runtime_script, get_percent_bucket_script, table, print_error
```